src/Model/FragmentInterface.php

```diff
@@ -67,4 +67,8 @@ public function getPosition(): ?int;
     public function getBackofficeTitle(): string;
 
     public function setBackofficeTitle(string $backofficeTitle): void;
+
+    public function setArticle(ArticleInterface $article = null): void;
+
+    public function getArticle(): ?ArticleInterface;
```

Comment:
Adding methods to an interface is a backwards compatibility break, because if will produce a crash for any implementation that does not have these methods.

Reply:
You could use `@method` on the interface to show, that these are new methods for the next major release.

Reply:
Is adding @method really enough or do we need to trigger a deprecation too?

Reply:
AFAIK there is no project that triggers a ``You should implement the new XY method`` deprecation.